src/libguac/tests/mem/realloc_or_die.c

```diff
@@ -0,0 +1,98 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include <CUnit/CUnit.h>
+#include <guacamole/mem.h>
+#include <stdint.h>
+
+/**
+ * Test which verifies that guac_mem_realloc_or_die() returns NULL for all
```

Comment:
I don't suppose theres's any way to test that the "or die" part works successfully?

Reply:
Hm ... I suppose we could try `fork()`ing and verifying that the process exits as expected. I'll give it a shot.

Reply:
Yep - that seems to work well.

Looks like I forgot to add `*_or_die()` tests for the other arithmetic operations. I'll do that, too.

Reply:
Test for the `*_or_die()` variants of other arithmetic ops has now been added via rebase.